core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -854,6 +874,30 @@ object Parser extends ParserInstances {
   def flatMap01[A, B](pa: Parser[A])(fn: A => Parser1[B]): Parser1[B] =
     Impl.FlatMap1(pa, fn)
 
+  /** Replaces parsed values with the given value.
+    */
+  def as[A, B](pa: Parser[A])(b: B): Parser[B] =
+    map(Impl.Void(pa))(_ => b)
+
+  /** Replaces Parser1 parsed values with the given value.
+    */
+  def as1[A, B](pa: Parser1[A])(b: B): Parser1[B] =
+    map1(Impl.Void1(pa))(_ => b)
+
+  /** Apply Parser `n` times
+    */
+  def replicateA[A](n: Int, pa: Parser[A])(implicit
+      acc: Accumulator[A, List[A]]
+  ): Parser[List[A]] =
+    Impl.Replicate(pa, n, acc)
+
+  /** Apply Parser1 `n` times
+    */
+  def replicateA1[A](n: Int, pa: Parser1[A])(implicit
+      acc: Accumulator1[A, List[A]]
```

Comment:
I think we should just pass the List accumulator and not use an implicit here since it compilcates the call site but does not add power. What do you think?

Alternatively: we make this replicateAs and then allow us to replicate into different types (like Vector)